packages/resource-deployment/templates/dashboard.template.json

```diff
@@ -461,9 +461,9 @@
                                                             "resourceMetadata": {
                                                                 "id": "[variables('appInsightsResourceId')]"
                                                             },
-                                                            "name": "customEvents/custom/totalScanRequests",
+                                                            "name": "totalScanRequests",
                                                             "aggregationType": 1,
```

Comment:
Should these next three metrics use sum as well?

Reply:
It does use sum. What metrics do you refer particularly?
`"aggregationType": 1`

Reply:
Sorry I misread, thought these were using average